### ISSUE TITLE
Can't use the test from README

### ISSUE THREAD

**Issue by dsferruzza** [sun-opsys/api-doc#19]
Hi!

I'm trying to setup a test in order to run validation checks on the api docs.
I used the code in the `Notes` section of the README, but it doesn't compile :/

At first, I got an error because it couldn't find `controllers.ApiDocController.validate()` (I was using the controller from the README)
So I replaced it by this one: [app/controllers/ApiDocController.scala](https://github.com/sun-opsys/api-doc/blob/master/app/controllers/ApiDocController.scala)
Maybe it should be written to do so in the README, if it's the good thing to do.

Then, I got this error:
```
[error] [...]\test\ApiDocControllerSpec.scala:9: could not find implicit value for evidence parameter of type org.specs2.main.CommandLineAsResult[play.api.mvc.Result]
[error]         "Validate swagger api docs" in {
[error]                                     ^
```

Could you help me, please?

**Comment by kmatheussen** [comment]
Thank you. I guess the controller is not included in the package anymore, or that you have to write
```no.samordnaopptak.apidoc.controllers.ApiDocController.validate()```. I'll fix up that. The controller doesn't really do anything useful, so it shouldn't be included.

But to your question. "could not find implicit value for evidence parameter" seems to be specs2 way of saying that your test doesn't return anything. This is an unfortunate feature of specs2 I think, but it is easily solved just by adding "true" (for instance) at the end of the test, like this:
```
 "Validate swagger api docs" in {
     validate()
     true
}
```

**Comment by dsferruzza** [comment]
It compile, but it still doesn't work as expected :/

------

**app/controllers/ApiDocController.scala**
```scala
package controllers

import play.api._
import play.api.mvc._

import no.samordnaopptak.apidoc.ApiDoc
import no.samordnaopptak.apidoc.{AnnotationHelper, SwaggerUtil}
import no.samordnaopptak.json.J

object ApiDocController extends Controller {

  // The required swagger info object (see https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#infoObject)
  val swaggerInfoObject = J.obj(
    "info" -> J.obj(
      "title"   -> "Generated Swagger API",
      "version" -> "1.0"
    )
  )

  @ApiDoc(doc="""
    GET /api/v1/api-docs

    DESCRIPTION
      Get main swagger json documentation
  """)
  def get() = Action {
    val apidocs = AnnotationHelper.getApiDocsFromAnnotations()
    val generatedSwaggerDocs = SwaggerUtil.getMain("/", apidocs)
    val json = generatedSwaggerDocs ++ swaggerInfoObject
    Ok(json.asJsValue)
  }
}
```
*I replaced the `class` definition by an `object` so it compiles. Apart from that, it's the same as in the README.*

-------

**test/ApiDocControllerSpec.scala**
```scala
package test

import org.specs2.mutable.Specification
import play.api.test._
import play.api.test.Helpers._


class ApiDocControllerSpec extends Specification {
  "Validate swagger api docs" in {
    no.samordnaopptak.apidoc.controllers.ApiDocController.validate()
    true
  }
}
```

-------

And the runtime error I get when running the `test` task:
```
[myproject] $ test
[info] ApiDocControllerSpec
[info]
[error] ! Validate swagger api docs
[error]  There is no started application (Play.scala:71)
[error] play.api.Play$$anonfun$current$1.apply(Play.scala:71)
[error] play.api.Play$$anonfun$current$1.apply(Play.scala:71)
[error] play.api.Play$.current(Play.scala:71)
[error] no.samordnaopptak.apidoc.RoutesHelper$.getRouteEntries(RoutesHelper.scala:39)
[error] no.samordnaopptak.apidoc.controllers.ApiDocController$.validate$default$1(ApiDocController.scala:61)
[error] test.ApiDocControllerSpec$$anonfun$1.apply$mcZ$sp(ApiDocControllerSpec.scala:10)
[info]
[info]
[info] Total for specification ApiDocControllerSpec
[info] Finished in 404 ms
[info] 1 example, 0 failure, 1 error
[info]
[error] Error: Total 1, Failed 0, Errors 1, Passed 0
[error] Error during tests:
[error]         test.ApiDocControllerSpec
[error] (test:test) sbt.TestsFailedException: Tests unsuccessful
```

-----

I'm using Play 2.4.3 with Activator 1.3.6, by the way.

Any idea?

**Comment by kmatheussen** [comment]
Thanks for reporting. I've updated the README file now with working examples. If it still doesn't work, just look at the examples placed in app/controllers/*.scala and the tests for the examples in app/test/ApiDocControllerSpec.scala.

Note that I also changed the api a bit, so you'll have to update to 0.9.17:
```libraryDependencies += "com.github.sun-opsys" % "api-doc" % "0.9.17"```
(at least to use the examples in README.md)
